xapian-core/weight/tfidfweight.cc

```diff
@@ -214,13 +214,16 @@ TfIdfWeight::get_maxpart() const
 {
     Xapian::termcount wdf_max = get_wdf_upper_bound();
     Xapian::termcount len_min = get_doclength_lower_bound();
-    double wdfn = get_wdfn(wdf_max, len_min, len_min, wdf_max, wdf_norm_);
+    double wdfn = get_wdfn(wdf_max, len_min, len_min,
+			   wdf_max, wdf_norm_);
```

Comment:
This seems to be an unrelated formatting change (and not really a useful one either).

Reply:
Corrected.